README.org

```diff
@@ -9,6 +9,13 @@ Tested against Zabbix 3.0 with MySQL back end.
   - CentOS :: sudo yum install mysql-connector-python
   - Ubuntu :: sudo apt-get install python-mysql.connector
   - Additional Help :: If the package is not found see: http://dev.mysql.com/doc/connector-python/en/connector-python-installation-binary.html for more details.
+**** wavefront-sdk-python
+Install wavefront-sdk-python
```

Comment:
This is the only pre-requisite package for the wavefront-sdk-python. (Since I was making using of the containerised version of Ubuntu, I was having minimal packages and certainly installed the python-dev package)